server/controller.js

```diff
@@ -84,10 +84,10 @@ export default class Controller
 		this.updateAllPlayersHands(playersHands);
 	}
 
-	async updateAllPlayersHands(playersHands)
+	async updateAllPlayersHands(playersHands, name)

```

Comment:
we shouldnt be passing an individual users name into a method which does stuff with all players' hands data. And then this isnt used within the method, so i suspect this was something temporary you didnt clean up. Remove please